share/wake/lib/system/path.wake

```diff
@@ -96,33 +96,17 @@ export def basename (file: String): String =
 export def dirname (file: String): String =
     simplify "{file}/.."
 
-export def isPathDir: Path => Boolean = match _
-  BadPath _ = False
-  Path str = matches `0+` (hashcode str)
+export def isPathDir (path: Path): Boolean =
+    getPathHash path
+    | matches `0+`
 
 # Paths differ from Strings in that they have been hashed; their content is frozen
-from internals_wake export type Path
-from internals_wake import def Path BadPath # Path constructor stays private!
+export data Path =
```

Comment:
Please remove the `export`. We don't want the user able to construct `Path`.

Reply:
Ah, I must not have been looking when copying things out of `internals_wake`.  This should definitely not be exported!